src/qumin/cli.py

```diff
@@ -11,17 +11,19 @@
 
 log = logging.getLogger()
 
+
 @hydra.main(version_base=None, config_path="config", config_name="qumin")
 def qumin_command(cfg):
     log.info(cfg)
     md = Metadata(cfg, __file__)
 
     if cfg.patterns is None or cfg.action == "patterns":
-        overab = cfg.pats.overabundant == False
+        overab = cfg.pats.overabundant is False
```

Comment:
Can we call these `not_overab` and `not_defect` ? Or integrate them directly into the assertion ? My initial wording (overab holding the false value) is confusing.